### ISSUE TITLE
Labels should be truncated

### ISSUE THREAD

**Issue by roderik** [VictoriaMetrics/helm-charts#931]
I was trying to install the cluster chart, and it died during install because of 

```
Pod observability-vm-75ff870e-victoria-metrics-cluster-vmstorage-0 in StatefulSet observability-vm-75ff870e-victoria-metrics-cluster-vmstorage failed error: Pod "obse ││ rvability-vm-75ff870e-victoria-metrics-cluster-vmstorage-0" is invalid: metadata.labels: Invalid value: "observability-vm-75ff870e-victoria-metrics-cluster-vmstorage-64d9d4dbd9": must be no more than 63 characters
```

**Comment by Haleygo** [comment]
related https://github.com/VictoriaMetrics/helm-charts/pull/687#pullrequestreview-1643368066

**Comment by AndrewChubatiuk** [comment]
made changes to k8s-stack labels for dashboards and rules in #1212. it should be available in the next release

**Comment by AndrewChubatiuk** [comment]
hey @ToroNZ . this issue was fixed in release [v0.25](https://github.com/VictoriaMetrics/helm-charts/releases/tag/victoria-metrics-k8s-stack-0.25.0)
